questions/qa-date-format.en.html

```diff
@@ -123,29 +188,30 @@ <h3>Option Three: Use the Accept-Language HTTP header</h3>
       <li>Displaying a generated date in a Japanese date format such as <span lang="ja">２００３年４月２日</span> in an English page
         would probably look out of place.</li>
     </ul>
-    <p>How this is done will vary depending on your development environment. Here are some pointers for some common environments.</p>
-    <div class="example">
-      <h4>Java/JSP</h4>
-      <p>Call the <code>getLocale</code> method of the <code>ServletRequest</code> or <code>HttpServletRequest</code> object. Use the
-        returned <code>Locale</code> object to call <code>DateFormat</code>. Note that the SHORT format uses only numbers. If you want unambiguous formats
-        use FULL. In some locales even LONG is all numeric. <!--(<a href="http://www.honomichl.com/java.html">Sample code</a>)--></p>
-      <p>See also <a class="print" href="http://icu-project.org/">ICU4J</a> since it contains   more up-to-date data (and more functionality) than the JDK routines.</p>
-    </div>
-    <div class="example">
-      <h4>ASP</h4>
-      <p>Use <code>Request.ServerVariables('HTTP_ACCEPT_LANGUAGE')</code> to get the user's preferences. Parse the first locale from the list
-        of accepted locales. You'll have to do your own mapping from the alphabetic locale code to a numeric Locale Identifier. Set <code>Session.LCID</code> to the resulting value. Call <code>FormatDateTime</code> to format the date.</p>
-      <p>Use vbLongDate to avoid ambiguity.</p>
-    </div>
-    <div class="example">
-      <h4>Perl</h4>
-      <p>Use <code>$ENV{'HTTP_ACCEPT_LANGUAGE'}</code> to get the preferred language. Use <code>POSIX:strftime</code> to format date values.
-        You'll have to do your own mapping of the accepted languages value to a date format string.</p>
-    </div>
+    <p>How this is done will vary depending on your development environment.</p>
```

Comment:
The section "[Store and display a date](https://deploy-preview-327--i18n-drafts.netlify.app/questions/qa-date-format.en#display)" also seems to lack focus.  In fact, i think it's trying to do 2 separate things:
1. describe how local date/time formats can be different and problematic, and what works or doesn't work if you try to use a generic format
2. describe how to produce dates and times in your code (one method described is that using JavaScript – but it doesn't mention the time element in HTML

Is option 3 really something useful these days?  Or is it part of the discussion "How language interacts with date/time settings and where to get the language from" ?

Reply:
Do you have any specific suggestions? I think option 3 is still useful. Do you think it should be removed or split into a separate article?